src/main/Core/Autostart/WindowsStoreAutostartManager.ts

```diff
@@ -32,7 +34,12 @@ export class WindowsStoreAutostartManager implements AutostartManager {
 
         try {
             const shortcutLink = this.shell.readShortcutLink(shortcutFilePath);
-            return shortcutLink.target === this.process.execPath;
+            if (shortcutLink.target) {
+                return shortcutLink.target === this.shortcutTarget;
+            }
+
+            const shortcutFileContent = this.fileSystemUtility.readTextFileSync(shortcutFilePath, "utf-16le");
+            return shortcutFileContent.includes(this.appId);
```

Comment:
Is there a reason why we can't or shouldn't do this?

```typescript
return shortcutFileContent === this.shortcutTarget;
```

Reply:
In other words: why would the shortcut file content contain the `appId` but not be exactly the `shortcutTarget`?

Reply:
Because the file content looks like this (copy&paste does not work):

![grafik](https://github.com/user-attachments/assets/374c15b0-c06a-45ff-b9cb-3d604651879b)

The `shell:AppsFolder\` part is missing.